pkg/subctl/cmd/join.go

```diff
@@ -69,6 +69,7 @@ var (
 	submarinerDebug               bool
 	operatorDebug                 bool
 	labelGateway                  bool
+	loadBalancer                  bool
```

Comment:
```suggestion
	loadBalancerEnabled           bool
```
so the flag becomes --loadBalancerEnabled = true or false

Reply:
I applied your suggestion. Thanks :)